### ISSUE TITLE
Modified ATSS with ResNet-152 can't show the proper score.

### ISSUE THREAD

**Issue by dev-sungman** [open-mmlab/mmdetection#6760]
**Notice**

There are several common situations in the reimplementation issues as below

1. Reimplement a model in the model zoo using the provided configs
2. Reimplement a model in the model zoo on other dataset (e.g., custom datasets)
3. Reimplement a custom model but all the components are implemented in MMDetection
4. Reimplement a custom model with new modules implemented by yourself

There are several things to do for different cases as below.

- For case 1 & 3, please follow the steps in the following sections thus we could help to quick identify the issue.
- For case 2 & 4, please understand that we are not able to do much help here because we usually do not know the full code and the users should be responsible to the code they write.
- One suggestion for case 2 & 4 is that the users should first check whether the bug lies in the self-implemented code or the original code. For example, users can first make sure that the same model runs well on supported datasets. If you still need help, please describe what you have done and what you obtain in the issue, and follow the steps in the following sections and try as clear as possible so that we can better help you.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. The issue has not been fixed in the latest version.

**Describe the issue**

My environments as follows:
mmcv: 1.2.4
mmdetection : 2.9.0

I also checked in the other environments:
mmcv: 1.4.0
mmdetection : 2.19.0

In both environments, I can't get the proper test accuracy when I used the ATSS model with ResNet-152 modification. 
I checked my pre-trained ResNet-152 ATSS model in another repository (mAP: 46.2). 
However, I could got only 0.6 mAP in test phase by using mmdetection. 

All I modified in mmdetection is adding configuration for ResNet-152 ATSS for using ResNet-152 ATSS.
![image](https://user-images.githubusercontent.com/36710497/145698514-04cb8264-06d6-4496-8991-8bbde42a208f.png)

**Reproduction**

1. What command or script did you run?

 python tools/test.py configs/atss/atss_r152_fpn_1x_coco.py pre-trained_resnet152_atss.pth --eval=bbox

2. What config dir you run?

I added ResNet-152 ATSS by following ResNet-101 ATSS configuration. 
I attached my configuration as above.

3. Did you make any modifications on the code or config? Did you understand what you have modified?

Yes, I modified a configuration for using ResNet-152 ATSS. 

4. What dataset did you use?

I used the COCO testset. 

**Environment**

1. Please run `python mmdet/utils/collect_env.py` to collect necessary environment information and paste it here.

![image](https://user-images.githubusercontent.com/36710497/145698579-06a51504-d35a-4a0f-bbb3-d172b286a0a4.png)
![image](https://user-images.githubusercontent.com/36710497/145698580-f4d9ab85-dc04-445c-97c7-ee2bc4f398dd.png)

2. You may add addition that may be helpful for locating the problem, such as
   1. How you installed PyTorch [e.g., pip, conda, source]  --> conda install. 
   2. Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Results**
![image](https://user-images.githubusercontent.com/36710497/145698612-42524445-ae86-42a3-9696-a6f909e92d66.png)

**Issue fix**

**Comment by dev-sungman** [comment]
I checked the performance of the pre-trained weight  at https://github.com/openvinotoolkit/training_extensions

And below image is represented as the experimental result. 
![image](https://user-images.githubusercontent.com/36710497/145700334-51eddcd1-c1af-4fac-bd84-16edb9d42762.png)

and my pretrained model which I used : https://drive.google.com/file/d/14A6me0KKcTXWXfRB2H2aQn_NLUB06yXa/view?usp=sharing

**Comment by dev-sungman** [comment]
I am sorry. I found that the preprocessing method is different.
